types/node/cluster.d.ts

```diff
@@ -170,11 +170,14 @@ declare module "cluster" {
         prependOnceListener(event: "setup", listener: (settings: ClusterSettings) => void): this;
     }
 
+    const SCHED_NONE: number;
```

Comment:
should we add these two also to `interface cluster`?

Reply:
Trying to dedupe cluster, will take care of this tho!